spring-cloud-starter-stream-processor-grpc/README.adoc

```diff
@@ -0,0 +1,68 @@
+//tag::ref-doc[]
+= gRPC Processor
+
+This processor uses gRPC to process Messages via a remote process written in any language that supports gRPC. This
+pattern, commonly referred to as `sidecar` is intended to allow the Java app to handle the stream processing while
+the Polyglot sidecar application handles the business logic. The sidecar application must implement a
+link:https://grpc.io/[grpc] service using link:../grpc-app-protos/src/main/proto/processor.proto[this Google protocol
+buffers schema].
+
+[NOTE]
+====
+The gRPC client stub is blocking by default. An asynchronous stub is provided as an option. This might perform
+better under certain circumstances (e.g., if the server is multi-threaded) however message ordering will not be
+guaranteed.
+====
+
+== Python support
+For Python sidecar applications, the link:https://github.com/dturanski/springcloudstream[springcloudstream] package is
```

Comment:
Why do we have to include a personal repo? Should we instead port this to a sample and link that instead?

Reply:
The sample would have that link as well if that's ok

Reply:
Sounds good. I'll create a sample and update the README